### PR TITLE
Release crowdloan ws integration + minor fixes

### PR DESCRIPTION
**Pull request by alessioscalici**

### CODE REVIEW

File: src/components/ParachainCrowdloan/shared/context/AuctionContext.tsx [hunk 1]
```diff
@@ -127,14 +135,29 @@ export const AuctionContextProvider: React.FC = ({ children }) => {
...
+      )
+        .then((r) => r.json())
+        .then((data) => {
+          setTotalContributions(data.numberOfContributions);
+          const totalRaisedNum = new BigNumber(data.totalStaked)
+            .div(DOT_PLANCK)
+            .toNumber();
+          setTotalRaised(totalRaisedNum);
+          // console.log("total raised", totalRaisedNum);
+          // console.log("base reward rate", REWARDS_BUDGET / totalRaisedNum);
```
Comment (offerijns) [inline src/components/ParachainCrowdloan/shared/context/AuctionContext.tsx:151]:
```suggestion
```

File: src/components/ParachainCrowdloan/Content/TopContributors.tsx [hunk 2]
```diff
@@ -11,18 +10,19 @@ type TopContributorsResponse = {
...
     (async () => {
-      const response = await fetch("/.netlify/functions/getTopContributors", {
-        method: "POST",
-        body: JSON.stringify({ amount: 5, parachain: PARACHAIN_NAME }),
-      });
+      const response = await fetch(
+        `/.netlify/functions/getCentrifugeTopContributors?parachain=${PARACHAIN_NAME}&amount=${5}`,
+        {
+          method: "GET",
+          // body: JSON.stringify({ amount: 5, parachain: PARACHAIN_NAME }),
```
Comment (offerijns) [inline src/components/ParachainCrowdloan/Content/TopContributors.tsx:23]:
```suggestion
```

File: lambda/getCentrifugeRewardData.js [hunk 3]
```diff
@@ -0,0 +1,267 @@
...
+      .times(baseRewardRate.min)
+      .times(curConfig.REWARD_REFERRAL_PERCENT / 100),
+    cur: totalReferredDOT
+      .times(baseRewardRate.cur)
+      .times(curConfig.REWARD_REFERRAL_PERCENT / 100),
+  };
+
+  // const numberOfReferrals = allContributions.filter(({ referralCode }) =>
+  //   referralCodes.includes(referralCode)
+  // ).length;
```
Comment (offerijns) [inline lambda/getCentrifugeRewardData.js:240]:
```suggestion
```

**Review by offerijns — APPROVED** [review]

File: lambda/getCentrifugeRewardData.js (hunk 3 above)
Comment (alessioscalici) [reply lambda/getCentrifugeRewardData.js:240]: I commented that out because we are not using that piece of data, but we might want to add it :D

**Pull request merged**